manifests/nodejs.yml

```
@@ -41,7 +41,7 @@ refs:
   - &ref_5_25_0 '>=5.25.0 || ^4.49.0'
   - &ref_5_26_0 '>=5.26.0 || ^4.50.0'
   - &ref_5_27_0 '>=5.27.0 || ^4.51.0'
-  - &ref_5_29_0 '>=5.29.0 || ^4.53.0'  # express 5 support
+  - &ref_5_29_0 '>=5.29.0 || ^4.53.0'
```

give me back my comment
```suggestion
  - &ref_5_29_0 '>=5.29.0 || ^4.53.0' # express 5 support
```